Userland/Libraries/LibJS/Runtime/ErrorTypes.h

```diff
@@ -230,6 +230,7 @@
     M(StringNonGlobalRegExp, "RegExp argument is non-global")                                                                           \
     M(StringRawCannotConvert, "Cannot convert property 'raw' to object from {}")                                                        \
     M(StringRepeatCountMustBe, "repeat count must be a {} number")                                                                      \
+    M(StringRepeatCountTooBig, "repeat count too big")                                                                                  \
```

Comment:
These error strings are visible to the user, so let's be a bit more descriptive in the message. Something like:
```
M(StringRepeatCountMustNotOverflow, "repeat count must not overflow")
```
(which matches what other engines display on this error)